pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>com.devonfw</groupId>
     <artifactId>maven-parent</artifactId>
-    <version>9</version>
+    <version>11</version>
```

Comment:
honestly, shouldn't we start with java 17? Having java 8 as the standard generated stack sounds crazy especially if we want to start a "new age" with this archetype

Reply:
I am fine to release it as intermediate version, but I would strongly recommend to not propose it as the new age solution yet. First I would start a proper java upgrade

Reply:
This is the version of `maven-parent` and not the `java-version`.

Reply:
Agree, I just reviewed what java version is configured in maven-parent@11 and it's java 8 Oo